script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -61,12 +61,12 @@ Senpai's voice, coming from inside the room, is very lively.
 [sha:409b975a7e5963d648094b70b98dac02c0d160d2]{
 -- Page 99, Offset 22397.
 -- 何の準備をしているかは不明だけどかつてない明るさ、つまりライトを感じる。この扉をスライドさせた瞬間、天上からの光で廊下が真っ白になるぐらいの。
-I don't know what she's busy with, but she seems to give off a brightness, a radiance that I've never seen before from her. So much so that the moment I slide open the door, it would bathe the hallway in light like sunrays coming down from the Heavens.
+I don't know what she has her hands full with, but I feel a brightness――a radiance that I've never felt before. So much so that I feel like the moment I slide the door open, rays of light from heaven will paint the hallway in pure white.
```

Comment:
I feel like shiki has a very healthy dependence on ciel that won't prove to be problematic at all